easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1906-foss-2018b.eb

```diff
@@ -0,0 +1,37 @@
+name = 'OpenFOAM'
+version = 'v1906'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package.
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer,
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://sourceforge.net/projects/openfoam/files/%(version)s/']
+sources = [SOURCE_TGZ]
+patches = ['OpenFOAM-%(version)s-cleanup.patch']
+checksums = [
+    '15e38c2dc659b63753a0dd3dff913222cc46d6a40089a1b76973dd741145f61a',  # OpenFOAM-v1906.tgz
```

Comment:
@mboisson Incorrect checksum?

Should be `bee03c4b1da0d2c9f98eb469eeffbce3a8614728ef8e87f664042a7490976537` (see also #9364, confirmed with a fresh download)

Reply:
Hem... that's the checksum that was current when I installed it on our systems...

Reply:
So, after investigation, OpenFOAM re-released v1906 apparently. Mostly changes in comments, but a few changes in the code. 

```
[mboisson@build-node ~]$ diff -ru OpenFOAM-v1906.{old,new} | grep -- '^[-+][^-+#]' | grep -v "Released\|Copyright\|www.openfoam.com\|A nd\|blob\|F ield\|O peration\|M anipulation\|======\|\/\/"
+%------------------------------------------------------------------------------
+
+  You can see how your report will be rendered on the platform by using the
+  "preview" tab above
-OpenFOAM version :
-Operating system :
-Compiler         :
+  The "/label ~bug" text is a gitlab flag that will add the "bug" label to this
+  issue
-head=641c8ae55e13b527ed99fa30a7d660dedee95dcc
+head=cebdbdff0e71ba850b2d771fe9747184d0de7961
-    return HashPtrTable<IOobject>::filterKeys(pred, pruning);
+    label changed = 0;
+    for (iterator iter = begin(); iter != end(); ++iter)
+    {
+        if
+        (
+            (pred(iter.key()) ? pruning : !pruning)
+         && erase(iter)
+        )
+        {
+            ++changed;
+        }
+    }
+    return changed;
-                    hpTa += sigma.value()*(emissivity_*lambdaTa4 + pow4(Ta));
+                    hpTa += emissivity_*sigma.value()*(lambdaTa4 + pow4(Ta));
-                    hpTa += sigma.value()*pow4(Ta);
+                    hpTa += emissivity_*sigma.value()*pow4(Ta);
-    if (debug)
-    {
-        const scalar Q = gSum(kappa(Tp)*patch().magSf()*snGrad());
-        Info<< patch().boundaryMesh().mesh().name() << ':'
-            << patch().name() << ':'
-            << internalField().name() << " :"
-            << " heat transfer rate:" << Q
-            << " wall temperature "
-            << " min:" << gMin(*this)
-            << " max:" << gMax(*this)
-            << " avg:" << gAverage(*this)
-            << endl;
-    }
+    DebugInfo
+        << patch().boundaryMesh().mesh().name() << ':' << patch().name() << ':'
+        << internalField().name() << " :"
+        << " heat transfer rate:" << gSum(kappa(Tp)*patch().magSf()*snGrad())
+        << " wall temperature "
+        << " min:" << gMin(*this)
+        << " max:" << gMax(*this)
+        << " avg:" << gAverage(*this) << nl;
```

Reply:
Sneaky bastards... They should be quartered.